src/test/java/com/nerodesk/TkGetFileTest.java

```diff
@@ -68,13 +65,18 @@ public void returnsFileContent() throws Exception {
         final String path = "some_file.txt";
         final String content = "some text content";
         storage.put(path, IOUtils.toInputStream(content));
-        final Matcher matcher = Pattern.compile(TkGetFile.PATH)
-            .matcher(String.format("http://localhost:8080/api/file/%s", path));
-        matcher.find();
-        final RqRegex regex = Mockito.mock(RqRegex.class);
-        Mockito.when(regex.matcher()).thenReturn(matcher);
         MatcherAssert.assertThat(
-            IOUtils.toString(new TkGetFile(storage).route(regex).act().body()),
+            IOUtils.toString(
+                new TkGetFile(storage).route(
+                    new RqRegex.Fake(
+                        "/api/file/(?<path>[^/]+)",
```

Comment:
I don't see the point of this `/api/file/` prefix here. Why can't we just use `(?<path>.*)` here?